pkg/transport/http.go

```diff
@@ -102,8 +105,17 @@ func (t *SSETransport) Setup(ctx context.Context, runtime rt.Runtime, containerN
 	t.runtime = runtime
 	t.containerName = containerName
 
-	// Add transport-specific environment variables
-	envVars["MCP_TRANSPORT"] = "sse"
+	envMap := map[types.TransportType]string{
```

Comment:
Make this a package level variable so it does not need to be instantiated each time it's called.